packages/vuelidate/src/index.js

```diff
@@ -14,6 +14,22 @@ const VuelidateRemoveChildResults = Symbol('vuelidate#removeChiildResults')
  * @return {UnwrapRef<*>}
  */
 export function useVuelidate (validations, state, registerAs) {
+  if (!validations) {
+    const instance = getCurrentInstance()
+    if (instance.type.validations) {
+      const rules = instance.type.validations
+
+      state = computed(() => toRefs(reactive(instance.ctx)))
+      validations = computed(() => isFunction(rules)
+        ? rules.call(state.value)
+        : rules
+      )
+
+      const uid = instance.uid || instance._uid
+      registerAs = `_vuelidate_${uid}`
```

Comment:
We already do this below, might clean it up a bit.

Reply:
Done